pkg/schedule/schedulers/balance_leader.go

```diff
@@ -541,3 +539,18 @@ func (l *balanceLeaderScheduler) createOperator(solver *solver, collector *plan.
 	op.SetAdditionalInfo("targetScore", strconv.FormatFloat(solver.targetScore, 'f', 2, 64))
 	return op
 }
+
+// IsDiable implements the Scheduler interface.
+func (l *balanceLeaderScheduler) IsDisable() bool {
+	return l.conf.isDisable()
+}
```

Comment:
base_scheduler.go implemented it.

Reply:
If we want to implement it once, we need to put `isDisable` and `setDisable` into `schedulerConfig` interface. Do you think you need to do this?

Reply:
WDYT? Is it possible that we might customize it?

Reply:
In this pr, we only need to make a customized implementation for defaultScheduler. I think this is acceptable and only needs to be implemented four times.

If follow what you said, it does only need to be implemented once, but I feel that `setDisable` should not be placed in the `schedulerConfig` interface, because only `defaultScheduler` can be `setDisabled`. But if you insist, I can also modify it